requirements.txt

```diff
@@ -0,0 +1,5 @@
+colorama
```

Comment:
All packages need to version lock so we can be sure of a working environment when packages upgrade.

Reply:
With a brief search online, it looks like there are multiple ways to do this. Do you have a recommendation for a best practice to do this?

Reply:
it would be:

```
colorama=x.y.z
pexpect=x.y.z
etc.
```

You can get the currently installed versions with `pip list`